data/ecosystems/m/moonbeam.toml

```diff
@@ -19,43 +20,53 @@ sub_ecosystems = [
   "Cask",
   "Certhum",
   "Chaotic Finance",
+  "Chimoney",
   "Collectify",
   "Convergence Finance",
   "Cougar Swap",
   "COVA",
   "Cover Compared",
   "crystyl.finance",
+  "DAM Finance",
   "DefiYield",
   "Demodyfi",
   "Exiled Racers",
+  "Fidi",
   "Firefly",
   "Galxe",
   "Huckleberry",
   "IBetYou",
+  "Karma HQ",
   "Klever",
   "Kodadot",
+  "Lucky Friday",
   "Lunardex",
   "MidasProtocol",
   "Minterest Finance",
   "MixBytes",
   "Moonbeans",
   "Moonfarmin",
+  "Moonfit",
   "Moonsama",
   "Moonwell",
   "Mystiko",
   "MyTrade",
   "Narwhalswap",
   "Novasama Technologies",
   "OnFinality",
+  "Onramper",
```

Comment:
I only saw traces of Onramper using Moonbeam in that it allows swapping of Glimmer, anything else? If not then I think it could be removed

Reply:
Agreed, removed